docs/components/pagination/index.html

```diff
@@ -0,0 +1,144 @@
+---
+title: Pagination
+also:
+  components/buttons: Buttons
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <h2 class="hxSectionTitle" id="first-page">First Page</h2>
+    <div class="demo">
+      <div class="hxPagination hxBtnGroup">
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-start"></hx-icon>
+        </button>
+        <button class="hxBtn" disabled>
+          <hx-icon type="angle-left"></hx-icon>
+        </button>
+        <button class="hxBtn" aria-current="true">1</button>
+        <button class="hxBtn">2</button>
+        <button class="hxBtn">3</button>
+         <button class="hxBtn">4</button>
```

Comment:
Check indent; looks off in review mode:
![screen shot 2018-03-27 at 11 04 16 am](https://user-images.githubusercontent.com/1538692/37979610-a42381ce-31ae-11e8-9c77-d261b78b34d4.png)

Reply:
good catch -- fixed